packages/labs/gen-wrapper-vue/test-output/package.json

```diff
@@ -42,7 +42,8 @@
         "installSelf"
       ],
       "files": [
-        "src/**/*.ts, src/**/*.vue",
+        "src/**/*.ts",
+        "src/**/*.vue",
```

Comment:
Tried to investigate the test-local error that points to a test from this package.json. Locally was unable to repro.

Reply:
Actually, for failure in this particular pipeline, it might be that we have an erroneous cached build artifact. Unsure if there's a way to cache bust the build?